keyboards/lovelive9/config.h

```diff
@@ -0,0 +1,63 @@
+
+
+#ifndef CONFIG_H
```

Comment:
We can replace the include guards with `#pragma once`

```suggestion
#pragma once
```